src/messages/request.rs

```diff
@@ -47,6 +47,15 @@ pub enum Request {
     },
 
     // --- MutableData ---
+    /// Fetches whole MutableData from the network.
```

Comment:
Let's document that during churn, this is unlikely to accumulate. Also for the other bulk gets (keys and values), and also for the methods in `src/node.rs`.